spec/lib/whitehall_importer/integrity_checker_spec.rb

```diff
@@ -0,0 +1,144 @@
+# frozen_string_literal: true
+
+RSpec.describe WhitehallImporter::IntegrityChecker do
+  describe "#valid?" do
+    let(:edition) do
+      create(
+        :edition,
+        tags: {
+          primary_publishing_organisation: [SecureRandom.uuid],
+          organisations: [SecureRandom.uuid],
+        },
+      )
+    end
```

Comment:
Can we use build(:edition) rather than create so we don't need to hit the database?